docs/guide/node-views/react.md

```diff
@@ -96,6 +97,15 @@ Keep in mind that this content is rendered by Tiptap. That means you need to tel
 
 The `NodeViewWrapper` and `NodeViewContent` components render a `<div>` HTML tag (`<span>` for inline nodes), but you can change that. For example `<NodeViewContent as="p">` should render a paragraph. One limitation though: That tag must not change during runtime.
 
+## Changing the default content tag for a node view
+
+By default a NodeView rendered by `ReactNodeViewRenderer` will always have a wrapping `div` inside. If you want to change the type of this node, you can the `contentDOMElementTag` to the `ReactNodeViewRenderer` options:
```

Comment:
@bdbch I don't want to be too petty. Shall we write "node view" here instead of NodeView?

Reply:
done! 👍